juicy-checkbox-group-item.html

```diff
@@ -161,7 +161,7 @@
             </template>
             <template is="dom-if" if="{{!checkboxHidden}}">
                 <div id="checkboxItemContainer">
-                    <div id="checkboxItem" data-checkable$="{{checkable}}" data-checked$="{{checked}}" on-click="toggleCheckbox">
+                    <div id="checkboxItem" data-checkable$="{{checkable}}" data-checked$="{{checked}}" on-click="toggleCheckbox" on-keypress="keypressCheckbox" tabindex="0">
```

Comment:
Normally such things are done with `change` event. Have you tried that?

We even do it in KitchenSink: https://github.com/StarcounterApps/KitchenSink/blob/master/src/KitchenSink/wwwroot/KitchenSink/CheckboxPage.html#L7